README.md

```diff
@@ -72,57 +72,58 @@ The card works with entities from within the **sensor** domain and displays the
+Entities may be listed directly (as per `sensor.temperature` in the following example), or defined using
+properties of the Entity object detailed in the following table (as per `sensor.pressure` in the following example):
 
 | Name | Type | Default | Description |
 |------|:----:|:-------:|-------------|
-| entity | string | **required** | Entity id of the sensor.
-| name | string | optional | Set a custom display name, defaults to entity's friendly_name.
-| color | string | optional | Set a custom color, overrides all other color options including thresholds.
-| unit | string | optional | Set a custom unit of measurement, overrides `unit` set in base config.
-| show_state | boolean | optional | Display the current state.
-| show_indicator | boolean | optional | Display a color indicator next to the state, (only when more than two states are visible).
-| show_line | boolean | optional | Set to false to hide the line (see note below table).
-| show_fill | boolean | optional | Set to false to hide the fill (see note below table).
-| show_points | boolean | optional | Set to false to hide the points (see note below table).
-| show_legend | boolean | optional | Set to false to turn hide from the legend.
-| state_adaptive_color | boolean | optional | Make the color of the state adapt to the entity color.
-| y_axis | string | optional | If 'secondary', displays using the secondary y-axis on the right.
-| fixed_value | boolean | optional | Set to true to graph the entity's current state as a fixed value instead of graphing its state history.
+| entity ***(required)*** | string |  | Entity id of the sensor.
+| name | string |  | Set a custom display name, defaults to entity's friendly_name.
+| color | string |  | Set a custom color, overrides all other color options including thresholds.
+| unit | string |  | Set a custom unit of measurement, overrides `unit` set in base config.
+| show_state | boolean |  | Display the current state.
+| show_indicator | boolean |  | Display a color indicator next to the state, (only when more than two states are visible).
+| show_line | boolean |  | Set to false to hide the line (see note below table).
+| show_fill | boolean |  | Set to false to hide the fill (see note below table).
+| show_points | boolean |  | Set to false to hide the points (see note below table).
+| show_legend | boolean |  | Set to false to turn hide from the legend.
+| state_adaptive_color | boolean |  | Make the color of the state adapt to the entity color.
+| y_axis | string |  | If 'secondary', displays using the secondary y-axis on the right.
+| fixed_value | boolean |  | Set to true to graph the entity's current state as a fixed value instead of graphing its state history.
 
 Note:
-- If the line and points and fill are all set to false, nothing will be visible in the graph. However, the
+- If the line, points and fill are all set to false, nothing will be visible in the graph. However, the
```

Comment:
After reading through your changes I noticed that this statement is outdated and is no longer the case, mind removing it while you're at it?

Reply:
Done